plugins/slack_webhooks/app/models/slack_webhook_notification.rb

```diff
@@ -17,7 +17,8 @@ def buddy_request(message)
 
   def default_buddy_request_message
     project = @deploy.project
-    ":pray: @here _#{@deploy.user.name}_ is requesting approval" \
+    # https://api.slack.com/docs/message-formatting
+    ":pray: <!here> _#{@deploy.user.name}_ is requesting approval" \
```

Comment:
This does not seem to work on the Slack desktop client. It is showing verbatim `<!here>`. Might be due to html encoding of `<` `>`.